javascript/node/selenium-webdriver/lib/webdriver.js

```diff
@@ -71,6 +71,19 @@ class Condition {
   description() {
     return this.description_;
   }
+
+  /**
+   * Allows for lenient instanceof checks
+   * @param {!(IThenable<T>|
+   *           Condition<T>|
+   *           function(!WebDriver): T)} condition - the condition instance
+   * @returns {*|boolean}
+   */
+  static [Symbol.hasInstance](condition) {
+    return condition
```

Comment:
`return !!condition`

Reply:
Actually, I'd probably be a little stricter:

`return !!condition && typeof condition === 'object'`

(plus the next two lines)